ui/package.json

```diff
@@ -55,6 +55,7 @@
     "test": "./node_modules/.bin/karma start --single-run --browsers Chrome"
   },
   "dependencies": {
-    "jasmine-core": "^2.5.2"
+    "jasmine-core": "^2.5.2",
+    "package.json": "^2.0.1"
```

Comment:
Hi @tanmayanand44, What this line is about? Can you explain what you did in this PR?

Reply:
I ran "yarn add package.json" to  install all the dependencies  to package.json.

Reply:
Oh, no, That's not how it is done. Did you go through the migration guide given in the warning message in the issue? https://bower.io/blog/2017/how-to-migrate-away-from-bower/

Reply:
Yes,read it.But got very little how to implement.
Can u guide me a little.

Reply:
Yeah, sure! 

First, install Yarn on your machine. Then navigate to the `ui` folder of Bassa and issue the following commands.
```
yarn global add bower-away
bower-away
```

Then run `bower-away --apply` and then `rm -rf bower_components`. Finally, run `yarn`. 

You might get some errors while Yarn is installing the dependencies. That's okay, commit your changes, push to your repo and create a PR. Comment the issues you got while Yarn install.  I'll tell you how to fix them then are there.

But start with a clean branch checkout from `master` ( `git checkout -b 253-bower-migration master ).  Don't include irrelevant commits in the PR like `Tanmay Anand Contribution`.

Reply:
when executing bower-away command it shows  'command not found'.

Reply:
All the commands worked except bower-away and bower-away --apply

Reply:
You ran `yarn global add bower-away` right? Did it install correctly?

Reply:
Yes it it did.

Reply:
Hmm, maybe give it a try again after restarting the machine?